sdks/unity/AgonesSdk.cs

```diff
@@ -58,6 +58,8 @@ private struct KeyValueMessage
         // Use this for initialization.
         private void Start()
         {
+            String port = Environment.GetEnvironmentVariable("AGONES_SDK_HTTP_PORT");
+            sidecarAddress = "http://localhost:" + (port != null ? port : "59358");

```

Comment:
you can replace it with ?? 
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator

Reply:
Sweet. I've updated it.